x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/rules_stats_collector.ts

```diff
@@ -65,141 +73,153 @@ const getRulesStatsQuery = (): SearchRequest => ({
+                  '@timestamp': 'desc',
+                },
+              },
             },
-            size: 1,
-            sort: {
-              '@timestamp': 'desc',
+            posture_type: {
+              top_metrics: {
+                metrics: {
+                  field: 'rule.benchmark.posture_type',
+                },
+                size: 1,
+                sort: {
+                  '@timestamp': 'desc',
+                },
+              },
             },
-          },
-        },
-        benchmark_version: {
-          top_metrics: {
-            metrics: {
-              field: 'rule.benchmark.version',
+            rule_number: {
+              top_metrics: {
+                metrics: {
+                  field: 'rule.benchmark.rule_number',
+                },
+                size: 1,
+                sort: {
+                  '@timestamp': 'desc',
+                },
+              },
             },
-            size: 1,
-            sort: {
-              '@timestamp': 'desc',
+            benchmark_id: {
+              top_metrics: {
+                metrics: {
+                  field: 'rule.benchmark.id',
+                },
+                size: 1,
+                sort: {
+                  '@timestamp': 'desc',
+                },
+              },
             },
-          },
-        },
-        benchmark_name: {
-          top_metrics: {
-            metrics: {
-              field: 'rule.benchmark.name',
+            benchmark_version: {
+              top_metrics: {
+                metrics: {
+                  field: 'rule.benchmark.version',
+                },
+                size: 1,
+                sort: {
+                  '@timestamp': 'desc',
+                },
+              },
             },
-            size: 1,
-            sort: {
-              '@timestamp': 'desc',
+            benchmark_name: {
+              top_metrics: {
+                metrics: {
+                  field: 'rule.benchmark.name',
+                },
+                size: 1,
+                sort: {
+                  '@timestamp': 'desc',
+                },
+              },
             },
-          },
-        },
-        passed_findings_count: {
-          filter: {
-            bool: {
-              filter: [
-                {
-                  bool: {
-                    should: [
-                      {
-                        term: {
-                          'result.evaluation': 'passed',
-                        },
+            passed_findings_count: {
+              filter: {
+                bool: {
+                  filter: [
+                    {
+                      bool: {
+                        should: [
+                          {
+                            term: {
+                              'result.evaluation': 'passed',
+                            },
+                          },
+                        ],
+                        minimum_should_match: 1,
```

Comment:
nit, this code works but looking at https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-bool-query.html
it looks like boolean filters dont need this much nesting to work. maybe worth giving it a try and clean this up a bit if we got time for that